src/components/commentsList/CommentsList.tsx

```diff
@@ -151,7 +151,10 @@ export function CommentsList({
 
   // Sort comments
   const computedComments = useMemo(
-    () => filteredComments.sort((a, b) => (a.createdAt > b.createdAt ? -1 : 1)),
+    () =>
+      [...filteredComments].sort((a, b) =>
```

Comment:
Ah, yeah because `.sort` updates the input! You can also use the new `.toSorted` Array function to avoid that! 
So you-re not destructuring every single time, and should be a little more performant!

Great catch! 

Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted